packages/jellyfish-transaction-builder/src/txn/txn_builder_governance.ts

```diff
@@ -38,7 +43,16 @@ export class TxnBuilderGovernance extends P2WPKHTxnBuilder {
         'CreateVoc address stack should be empty'
       )
     }
-    const creationFee = this.network.name === 'regtest' ? new BigNumber('5') : new BigNumber('50')
+    let creationFee = new BigNumber('5')
```

Comment:
I guess it's the same for these because the fees are implied (edited by the method) but not explicitly defined. Users might not be signing what they initially thought they explicitly defined.